action.sh

```diff
@@ -7,9 +7,9 @@ echo '🔧 Install tools'
 npm init -y && npm install -y postcss-cli autoprefixer
 
 echo '🤵 Install Hugo'
-HUGO_VERSION=$(curl -s https://api.github.com/repos/gohugoio/hugo/releases/latest | jq -r '.tag_name')
+HUGO_VERSION='v0.109.0'
```

Comment:
Why is this version pinned here?

Reply:
My bad, didn't realize all future commits to the branch would get added to the pull request. Reverted, thanks! :)